src/views/LoginView.vue

```diff
@@ -1,9 +1,61 @@
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import BorderedButton from '@/components/Controls/BorderedButton.vue'
+import PrimaryButton from '@/components/Controls/PrimaryButton.vue'
+</script>
 
 <template>
-  <div>
-    <h1>Login</h1>
+  <div
+    style="height: calc(100vh - 56px)"
+    class="flex items-center justify-center bg-background-secondary"
+  >
+    <div class="w-200 rounded-2xl bg-background-primary px-14 py-10">
+      <h1 class="mb-5 h-9 font-primary text-28px font-medium text-[#1E1E1E]">ログイン</h1>
```

Comment:
詳しくは36行目への指摘を見てほしいのですが、family・size・weightに関しては`.fontstyle-ui-title`を使用していただけると簡潔だと思います。
```suggestion
      <h1 class="mb-5 fontstyle-ui-title text-[#1E1E1E]">ログイン</h1>
```
(なお、このような文字のスタイル名(`fontstyle-hoge-fuga`)は、Figma上でテキストをクリックすると楽に確認できます!)